### ISSUE TITLE
[Suggestion] Chest Customization

### ISSUE THREAD

**Issue by Lmmb74** [tastybento/askygrid#6]
Would you mind adding a sort of config for the chest randomization?

**Comment by tastybento** [comment]
Yes, this is a good idea. It won't be in 5.0, but it makes sense to allow this somehow.

**Comment by Lmmb74** [comment]
5.1?

**Comment by Lmmb74** [comment]
Btw, when I tried to compile the 5.0 jar (I'm testing 4.4 right now with a friend of mine), I got many errors from the class SkyGridGen.
